configure.php

```diff
@@ -1089,6 +1041,69 @@ function xinclude_residual( DOMDocument $dom )
     }
     if ( $see )
         echo "  See: https://github.com/php/doc-base/blob/master/docs/structure.md#xmlid-structure\n";
+
+    global $ac;
+    $fatal = $ac['LANG'] == 'en';
+
+    if ( $see && $fatal )
+        errors_are_bad( 1 ); // Duplicated strucutral xml:ids are fatal on doc-en
+}
+
+function xinclude_residual_list( DOMDocument $dom ) : DOMNodeList
+{
+    $xpath = new DOMXPath( $dom );
+    $xpath->registerNamespace( "xi" , "http://www.w3.org/2001/XInclude" );
+    $nodes = $xpath->query( "//xi:include" );
+
+    return $nodes;
+}
+
+function xinclude_debug_report( DOMDocument $dom )
+{
+    $debugFile = __DIR__ . "/temp/xinclude-debug.xml";
+
+    dom_saveload( $dom , $debugFile ); // preserve state
+
+    libxml_clear_errors();
+    $dom->xinclude();
+    $errors = libxml_get_errors();
+    libxml_clear_errors();
+
+    dom_saveload( $dom );              // normal output
+
+    $count = 0;
+    $prefix = realpath( __DIR__ );
+
+    $prevLine = -1;
+    $prevClmn = -1;
```

Comment:
```suggestion
    $prevColumn = -1;
```
Please don't omit vowels.

Reply:
The entire function is planned to be removed.